targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_SDK_11/libraries/util/nrf_assert.c

```diff
@@ -40,7 +40,7 @@
 #include "app_error.h"
 #include "nordic_common.h"
 
-#if defined(DEBUG_NRF)
+#if defined(DEBUG_NRF_USER)
```

Comment:
I guess it should be `#if defined(DEBUG_NRF) || defined(DEBUG_NRF_USER)`?

Reply:
Looks like this way it is used in nordic files as well.

Reply:
Thanks, [done](https://github.com/ARMmbed/mbed-os/compare/51fe2c2ccd258e480bfd3aae75c7ebc2e4152388..dfe0a58581e19f4a925410f8ae794929d9efeb0f).